#Message for nudging a joint clockwise, or counter clockwise
#useful for teleoping specific joints back into position
#joint name to nudge
string joint_name
#true if it should be nudged clockwise
bool clockwise
#amount to nudge it in radians
float64 amount
